pymc/distributions/continuous.py

```diff
@@ -415,7 +415,7 @@ class HalfFlatRV(RandomVariable):
     _print_name = ("HalfFlat", "\\operatorname{HalfFlat}")
 
     @classmethod
-    def rng_fn(cls, rng, size):
+    def rng_fn(cls, rng, *args):
```

Comment:
same here

Reply:
Same coment as above. We should keep `size` here

Reply:
```suggestion
    def rng_fn(cls, rng, size):
```